Prose/ProseLib/Sources/ProseCoreTCA/OrderedCollections+Ext.swift

```diff
@@ -0,0 +1,43 @@
+//
+// This file is part of prose-app-macos.
+// Copyright (c) 2022 Prose Foundation
+//
+
+import OrderedCollections
+
+public extension Dictionary {
+  @discardableResult mutating func prose_toggle<Element>(
```

Comment:
We have now 5 public, untested, (4) undocumented `prose_toggle` extensions and one caller. To me that begs the question if we shouldn't just inline the implementation in that one place where it is used.

Reply:
I don't know. It could almost be generic, but I found no way not to duplicate the code.
Maybe I can remove some non-ordered extensions at least, which we do not use.